tgui/packages/tgui/interfaces/BorgHypo.tsx

```diff
@@ -0,0 +1,87 @@
+import { toFixed } from 'common/math';
+import { useBackend } from '../backend';
+import { Button, Flex, NoticeBox, Section, ProgressBar } from '../components';
+import { Window } from '../layouts';
+
+type BorgHypoContext = {
+  maxVolume: number,
+  theme: string,
+  reagents: Reagent[],
+  selectedReagent: string;
+}
+
+type Reagent = {
+  name: string;
+  volume: number;
+  description: string;
+}
+
+export const BorgHypo = (_, context) => {
+  const { data } = useBackend<BorgHypoContext>(context);
+  const {
+    maxVolume,
+    theme,
+    reagents,
+    selectedReagent,
+  } = data;
+  return (
+    <Window
+      width={450}
+      height={(reagents.length * 25) + 60}
+      theme={theme}
+    >
+      <Window.Content>
+        <Section>
+          <ReagentDisplay
+            reagents={reagents}
+            selected={selectedReagent}
+            maxVolume={maxVolume} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const ReagentDisplay = (props, context) => {
+  const { act } = useBackend(context);
+  const { reagents, selected, maxVolume } = props;
+  if (reagents.length === 0) {
+    return (
+      <NoticeBox>
+        No reagents available!
+      </NoticeBox>
+    );
+  }
+  return reagents.map(reagent => (
+    <Flex key={reagent.name} m={0.5}>
```

Comment:
was there some reason to use Flex over Stack?

Reply:
Not particularly.